packages/esm-patient-tests-app/src/test-results/panel-view/panel-view.component.tsx

```diff
@@ -193,15 +209,15 @@ const PanelViewHeader: React.FC<PanelViewHeaderProps> = ({
     <div className={styles.panelViewHeader}>
       {!showSearchFields ? (
         <>
-          <div className={styles.flex}>
```

Comment:
Got rid of the default "Panel" text header to make it look more like the desktop view (which has no such header).